CompanyEmployees/MappingProfile.cs

```diff
@@ -14,6 +14,10 @@ public MappingProfile()
 
             CreateMap<CompanyForCreationDto, Company>();
             CreateMap<CompanyForUpdateDto, Company>();
+
+            // Attendance mappings
```

Comment:
Consider wrapping the attendance mappings with a region or comment block to keep entity-specific mappings clearly grouped. This helps with maintainability and readability as the number of mappings grows.

**Suggested change**
```suggestion
// Attendance mappings
// -------------------
CreateMap<Attendance, AttendanceDto>();
CreateMap<AttendanceForCreationDto, Attendance>();
```